Makefile

```diff
@@ -177,9 +177,9 @@ out/cf-cli_osx: $(GOSRC)
 	GOARCH=amd64 GOOS=darwin go build \
 				 -a -ldflags "$(LD_FLAGS)" -o out/cf-cli_osx .
 
-out/cf-cli_osx_arm: $(GOSRC)
```

Comment:
👍

Reply:
After a little bit of digging, I saw that in our current releases, we are generating the files `cf8-cli_8.7.10_macosarm.tgz` with the name `macosarm`, so this change makes sense in that context.

My question here is, should we change the binary name that we are creating, or should we keep it as is and fix the makefiles? @a-b @moleske

Reply:
no strong feelings.  if we decide to fix the make file I'd make a separate pr for that as it is not related to `Remove Content-Type header`

I think it is simpler to change the makefile over changing the binary name cause I suspect a lot of things depend on that binary name

Reply:
I don't understand why Makefile changes are there here. It was a typo on v8. But I ran the linter and killed it midway since it was taking up a lot of resources. Have removed the changes